CONTRIBUTING.md

```diff
@@ -1,182 +1,187 @@
+1. [Fixing things](#fixing-things)
+    - [Reporting Issues](#reporting-issues)
+    - [Submitting Pull Requests](#submitting-pull-requests)
+    - [Stale Issues and Pull Requests](#stale-issues-and-pull-requests)
+    - [Labels](#labels)
 1. [License](#license)
 
-## Git and GitHub
-The A11Y Project is an Open Source project. It uses [Git](https://git-scm.com/) and [GitHub](http://github.com/) to handle coordinating maintenance of the site and the publishing of new content and features.
 
-People working on more sophisticated aspects of the site (new features, bug fixes, etc.) are expected to have at least some familiarity with version control, GitHub's key functionality, and using [feature branches to work locally](#set-up-locally).
+## About this site
+
+Following is information about how the site operates. Instructions on how to get the site running locally on your computer can be found in the [`README`](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/README.md).
+
+### Git and GitHub
+The A11Y Project is an [Open Source](https://en.m.wikipedia.org/wiki/Open-source_software) project. It uses [Git](https://git-scm.com/) and [GitHub](http://github.com/) to handle publishing new content and features, as well as coordinating maintenance.
+
+People working on more sophisticated aspects of the site (new features, bug fixes, etc.) are expected to have at least some familiarity with version control, GitHub's key functionality, and [using feature branches to work locally](#set-up-locally).
 
 If you are looking for a place to get started learning about Git and contributing to an Open Source project, [Egghead.io offers a great introductory course](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github).
 
-## Pattern Submissions
-If you'd like to submit a [Pattern](https://a11yproject.com/patterns) make sure to do the following:
+### Build Process
+This project uses [Jekyll](https://jekyllrb.com/) to convert project files into a website that a browser can read and display. You'll need to make sure you've run all the commands in our [`README`](https://github.com/a11yproject/a11yproject.com#local-development) to get Jekyll working.
+
+Any changes you save will cause the local version of the site to recompile. You can then refresh the browser tab that has your local instance of the site open to preview your edits.
+
+### Important Branches
+
+`gh-pages` is the [production branch](https://www.atlassian.com/git/tutorials/using-branches). This is the live website.
+
+If you have a feature request, we suggest first [filing an Issue](#reporting-issues) to discuss it. Once that feature has been approved you can start coding! Create a new [feature branch](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) from `gh-pages`, and name it something that references the feature you'll be working on. For example a feature to increase the site's type size could be: `increase-font-size`.
```

Comment:
i'd expect the "filing an issue" link to go to the new issue screen.  maybe do that and then do a (read: 'reporting issues') where that's the link?